Packs/AutofocusFeed/Integrations/AutofocusFeed/AutofocusFeed_test.py

```diff
@@ -0,0 +1,28 @@
+from AutofocusFeed import Client
+
+INDICATORS = [
+    "d4da1b2d5554587136f2bcbdf0a6a1e29ab83f1d64a4b2049f9787479ad02fad",
+    "19.117.63.253",
+    "domaintools.com",
+    "flake8.pycqa.org/en/latest",
+    "19.117.63.253/28",
+    "2001:db8:85a3:8d3:1319:8a2e:370:7348",
+    "2001:db8:85a3:8d3:1319:8a2e:370:7348/32"
+]
+
+TYPES = [
```

Comment:
those are all the types possible from those two feeds?

Reply:
yes.
custom feed is a type of feed - it can be of any kind and chosen by the user.
daily feed is a mixed feed of all types